### ISSUE TITLE
Allow printable definition in token rule

### ISSUE THREAD

**Issue by SchrodingerZhu** [BNFC/bnfc#341]

**Comment by andreasabel** [comment]
> Add `printable` to represent all printable unicode chars

Yes, this would be great to have, but I don't know how to implement this.  `printable` would have to be mapped to the lexer generators our backends support, but I am not aware that e.g. Alex supports this, the issue https://github.com/simonmar/alex/issues/126 is still open.

See also #23.
